Formula/gitlab-ci-multi-runner.rb

```diff
@@ -4,8 +4,8 @@ class GitlabCiMultiRunner < Formula
   desc "The official GitLab CI runner written in Go"
   homepage "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner"
   url "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner.git",
-    :tag => "v1.5.2",
-    :revision => "76fdacdf55c5a83249a4bdc5bd8c650e7a5ff796"
+    :tag => "v1.5.3",
+    :revision => "fb49c4781e5040539d3faec87a2a63675d2c9f00"
```

Comment:
> But I think @DomT4 wants to make :branch an exception that never wraps, and I think possibly :shallow too, but I'm not sure.

It's not a hard rule. I'm fine with every `:` having its own line, I was just wary of bulking out formulae but eh.